client/src/pages/add-recipe/add-recipe-manual.page.js

```diff
@@ -74,27 +139,78 @@ const ManualInsertPage = () => {
                                         autoFocus
                                     />
                                 </Grid>
-                                <Grid item xs={12}>
-                                    <TextField
-                                        required
-                                        fullWidth
-                                        id="ingredients"
-                                        label="Ingredients"
-                                        name="ingredients"
-                                        multiline
-                                        rows={4}
-                                    />
+                                <Grid fullWidth item xs={12}>
+                                    {ingredients.map((field, index) => (
+                                        <div style={{ display: 'flex', justifyContent: 'center' }} key={index}>
+                                        <TextField
```

Comment:
This works great, good job! It may look a bit nicer if there is a little space between the input fields when new ones are added.